gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/common/impl/SimpleImportMappings.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.config.common.impl;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+
+import gobblin.config.store.api.ConfigStore;
+import gobblin.config.store.api.ConfigStoreWithResolution;
+import gobblin.config.store.api.VersionDoesNotExistException;
+
+
+/**
+ * Utility class to build the import mapping per {@ConfigStore} per version.
+ * 
+ * @author mitu
+ *
+ */
+public class SimpleImportMappings implements ImportMappings {
```

Comment:
I think the global cyclic checker should use an existing import mapping since it has the graph constructed already.